code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -1,5 +1,5 @@
 /obj/item/weapon/gun/projectile/automatic/pistol
-	name = "\improper FK-69 pistol"
+	name = "stechkin pistol"

```

Comment:
@Dorsisdwarf 
I'm 99% certain that it used to have an actual stetchkin sprite mport lifted from google images or some shit and scaled down like the deagle sprites

the resprite of everything probably made it a walther ppk